libs/eo/landing-page/shell/src/lib/eo-landing-page-shell.component.ts

```diff
@@ -35,20 +38,87 @@ const selector = 'eo-landing-page-shell';
   selector,
   styles: [
     `
+      @use '@energinet-datahub/watt/utils' as watt;
+
       ${selector} {
+        position: relative; // So we can use position absolute for the mesh illustrations at the top and bottom of the page
         display: block;
+
+        .${selector}__wrapper {
+          position: relative;
+          width: 100%; // This is used for styling 100% of the available screen width, with i.eg a background color
```

Comment:
suggestion: Remove `width` as we usually use block elements as layout elements, for example `<div>`.
```suggestion
```